### PR TITLE
Kb/root landing work

### PR DESCRIPTION
**Pull request by kburnell**

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/**](https://that-website-git-kb-root-landing-work.that-conference.now.sh/) |
|:-:|
|<a href="https://that-website-git-kb-root-landing-work.that-conference.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/9ou_ErP6ds_1e1aeyYNU_k6mKTI_.png">(view full size)</a> |







Commit 64e27eedbedbcb795bdc39f079ae470227fd209f (https://that-website-git-kb-root-landing-work.that-conference.now.sh).

File: components/Header/root.js [hunk 1]
```diff
@@ -31,6 +33,11 @@ const NavIcon = styled(Icon)`
   fill: ${({ theme }) => theme.colors.white};
   width: 5rem;
   padding: 0 1rem;
+
+  ${below.small`
```
Comment (saragibby) [inline components/Header/root.js:37]: Thanks for fixing this up!

File: pages/index.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Community from '../components/RootHomePage/Community';
 import Events from '../components/RootHomePage/Events';
 import Hero from '../components/RootHomePage/Hero';
 import Keynotes from '../components/RootHomePage/Keynotes';
-import LiveStream from '../components/RootHomePage/LiveStream';
+// import LiveStream from '../components/RootHomePage/LiveStream';
```
Comment (saragibby) [inline pages/index.js:9]: Was this giving you issues when you pulled in my branch?

File: components/RootHomePage/Keynotes.js [hunk 3]
```diff
@@ -1,12 +1,92 @@
...
+  );
+};
+
 const Keynotes = ({ className }) => {
   return (
-    <ContentSection className={className}>
-      <p>this is root keynote section</p>
-    </ContentSection>
+    <Container className={className}>
+      <ContainerWithBGImageAndLG
```
Comment (saragibby) [inline components/RootHomePage/Keynotes.js:65]: Can you use `ContentSection` for this now?

File: pages/index.js (hunk 2 above)
Comment (kburnell) [reply pages/index.js]: Nope commented it out as part of figuring out why there was horizontal scroll on mobile and forgot to uncomment it.

File: components/RootHomePage/Keynotes.js (hunk 3 above)
Comment (kburnell) [reply components/RootHomePage/Keynotes.js:65]: This is weird section where only the top piece has a full-width image.

**Pull request merged**